src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
```

Comment:
Do consider replacing such string literals with named constants, it will help in enhancing readability of your code